sdk.js

```diff
@@ -0,0 +1,3 @@
+const apiRouter = require("express").Router();
```

Comment:
What is this file? Any particular reason it's called `sdk`?

Reply:
I called it sdk because I followed what the other dev in the hackathon did. I guess I thought this was good practice when following documentation. But I can move it to app.js